src/main/java/com/neprozorro/model/LotItemInfo.java

```diff
@@ -0,0 +1,50 @@
+package com.neprozorro.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "lot_item_info")
+public class LotItemInfo {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "model")
+    private String model;
+
+    @Column(name = "price")
+    private BigDecimal price;
+
+    @Column(name = "amount")
+    private int amount;
+
+    @Column(name = "total_item_price")
+    private BigDecimal totalItemPrice;
+
+    @JsonBackReference
+    @ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
It's Eager by default, consider making in Lazy and adding join fetch to the custom query in the REpository

Reply:
How we discuss - in this case Lazy Fetch will be over-optimization for now.